drivers/interrupt_controller/intc_plic.c

```diff
@@ -73,8 +106,10 @@ static int riscv_plic_is_edge_irq(uint32_t irq)
  */
 void riscv_plic_irq_enable(uint32_t irq)
 {
+	const struct device *dev = get_plic_dev_from_irq(irq);
+	const struct plic_config *config = dev->config;
+	volatile uint32_t *en = (volatile uint32_t *) config->irq_en;
```

Comment:
will also need to use `sys_read32(..)` / `sys_write32(..)`

Please file a bug for that too.

Reply:
#62956